### DISCUSSION TITLE
v6 backward compatibility package for v5 timeline

### DISCUSSION THREAD

**Discussion by AngeloGiurano** [remix-run/react-router#8444] [Q&A] [answered]
Hi,

I have got simply a question since I'm trying to decide if to put in the effort of manually upgrading to v6 or wait for the backward compatibility package for v5.

Is there a timeline/date of when we could expect the package to be completed?

Thank you

**Comment by eglavin** [comment]
Yeah I'm trying to figure this out my self I also find the "Upgrading from v5" docs to be very confusing with how many examples are of the old methods as well might be worth thinking about having it cleaned up a bit?

**Comment by peter-coastpay-com** [comment]
did this package ever get made ?

**Comment by abernier** [comment]
I also had opened an issue about this, but it was closed despite any changes into the documentation...

For reference: https://github.com/remix-run/react-router/issues/8597

**Comment by ryanflorence** [comment, marked as answer]
The upgrade guide and instructions on how to use the compatibility package are here :)
https://github.com/remix-run/react-router/discussions/8753
